examples/daisyworld.jl

```diff
@@ -26,15 +28,25 @@
 # necessary for more white daisies to be produced in order to cool the temperature.
 # The interplay of the living and non living aspects of this world manages to find an
 # equilibrium over a wide range of parameter settings, although with enough external
-# forcing, the daisies will not be able to self regulate the temperature of the planet
+# forcing, the daisies will not be able to regulate the temperature of the planet
 # and eventually go extinct.
 
-# ## Defining the agent type
+# ## Defining the agent types
 
-# The agent here is not so complex. We see it has three values (other than the required
+# `Daisy` has three values (other than the required
 # `id` and `pos` for an agent that lives on a [`GridSpace`](@ref). Each daisy has an `age`,
 # confined later by a maximum age set by the user, a `breed` (either `:black` or `:white`)
 # and an associated `albedo` value, again set by the user.
+# `Land` represents the surface. We could make `Land` also have an albedo field, but
+# in this world all surface has the same albedo and thus we make it a model parameter.
+
+# Notice that the `Land` does not necessarily have to be an agent, and one could represent
+# surface temperature via a matrix (parameter of the model). This is done in an older version,
+# see file `examples/daisyworld_old.jl`. The old version has a slight performance advantage.
```

Comment:
Should we change this files name to be something like `daisyworld_matrix` or something? It's an older version sure, but for a new user that's not really relevant. What they're interested in is the choice between performance and usability.

Haven't confirmed, but my assumption is that the `_old` implementation will not show up in the docs now, but will stay in the examples directory for this link only. Is that right? If so - cool, that makes the most sense.